src/protocol/templates/Ping1D.py.in

```diff
@@ -49,30 +49,29 @@ class Ping1D(object):
         return None
 
     ## Write data to device
+    #  @return Number of bytes written
     def write(self, data):
         return self.iodev.write(data)
 
-    ## Make sure there is a device on the line and read some initial data
-    # Return True if the device replies with expected data
-    # Return False otherwise
+    ## Make sure there is a device on and read some initial data
+    #  @return True if the device replies with expected data, False otherwise
     def initialize(self):
         if self.request(PingMessage.PING1D_VOLTAGE_5) is None:
             return False
         return True
 
     ## Request the given message ID
-    # Return response if it is rxed within timeout period, None otherwise
-    # TODO handle nack to exit without blocking
+    #  @return PingMessage: the device reply if it is received within timeout period, None otherwise
```

Comment:
This documentation should have the parameters and the type, right ?

Reply:
yes